client/src/components/App.jsx

```diff
@@ -34,64 +40,84 @@ function App() {
+        <Grid>
+          <Header>Wheel of TBR</Header>
+          {showPromptInfo && (
+            <PromptInfo
+              triggerPrompt={triggerShowPromptInfo}
+              onClick={toggleShowPromptInfo}
+              {...{ prompts, colorObject, randomPageNumber }}
             />
-          ) : (
-            ''
           )}
-        </Prompt>
-
-        <WheelComponent winner={currentPrompt} {...{ mustSpin, setMustSpin }} />
-        <GridWrapper>
-          <Button
-            disabled={currentPrompt.includes(LAST_PROMPT) || mustSpin}
-            primary
-            autoFocus
-            onClick={onSpin}
-          >
-            Spin!
-          </Button>
-          <Button
-            disabled={currentPrompt.includes(INITIAL_PROMPT)}
-            onClick={onReset}
-          >
-            reset
-          </Button>
-        </GridWrapper>
-        {history.length ? (
-          <History history={history} onClick={toggleShowPromptInfo} />
-        ) : (
-          ''
-        )}
-      </Main>
-    </Grid>
+          <Main showPromptInfo={showPromptInfo}>
+            <Navigation />
+            <Switch>
+              <Route exact path="/">
+                <Prompt
+                  data-testid="prompt"
+                  {...(currentPrompt !== INITIAL_PROMPT &&
+                  currentPrompt !== LAST_PROMPT &&
+                  !mustSpin
+                    ? { onClick: toggleShowPromptInfo }
+                    : '')}
+                >
+                  {mustSpin ? <LoadingCircles /> : currentPrompt}
+                  {(!mustSpin && currentPrompt === 'Cover Colour') ||
+                  (!mustSpin && currentPrompt === 'Page Number')
+                    ? `:
+          `
+                    : ''}
+                  {(!mustSpin && currentPrompt === 'Cover Colour') ||
+                  (!mustSpin && currentPrompt === 'Page Number') ? (
+                    <PromptSpecifier
+                      {...{ currentPrompt, colorObject, randomPageNumber }}
+                    />
+                  ) : (
+                    ''
+                  )}
+                </Prompt>
+                <WheelComponent
+                  winner={currentPrompt}
+                  {...{ mustSpin, setMustSpin }}
+                />
+                <GridWrapper>
+                  <Button
+                    disabled={currentPrompt.includes(LAST_PROMPT) || mustSpin}
+                    primary
+                    autoFocus
+                    onClick={onSpin}
+                  >
+                    Spin!
+                  </Button>
+                  <Button
+                    disabled={currentPrompt.includes(INITIAL_PROMPT)}
+                    onClick={onReset}
+                  >
+                    reset
+                  </Button>
+                </GridWrapper>
+                {history.length ? (
+                  <History history={history} onClick={toggleShowPromptInfo} />
```

Comment:
I think you can use a better name here than a generic `onClick`. This way you only really know what should happen here and not in your component. Also `toggle` and `show` seem to indicate the same thing. 
```suggestion
                  <History history={history} onTogglePromptInfo={togglePromptInfo} />
```